### DISCUSSION TITLE
Passing mounts through from rootless podman to rootless podman

### DISCUSSION THREAD

**Discussion by tmstn** [containers/podman#23689] [General]
I have a rootless podman container running as `--privileged` that has another rootless podman container running inside of it. Is it safe to pass through bind mounts and mounted volumes (as bind mounts) through from the host into the inner container?

**Comment by rhatdan** [comment]
Well just as safe as it would be through to any container.  From an SELinux point of view their could be some limited risk in overlapping SELinux labels, if the SELinux labels on the inner container are even applied.  But other then this, I see no greater risk then running a regular container.

**Comment by tmstn** [comment]
Okay, thanks a lot :+1:
